### ISSUE TITLE
Console not logging JSON payload for specific element query

### ISSUE THREAD

**Issue by mslukebo** [microsoft/WinAppDriver#1438]
I have a test consistently failing to find an element via accessibility ID when its ran on a Azure Pipeline using the WinAppDriver Azure Pipeline task. When I run the test locally, it consistently passes. 

I inspected the console output of my local machine and the Pipeline task and zoomed into when the offending element is being queried. Here is my local output:

```
==========================================
POST /session/<session_id>/element HTTP/1.1
Accept: application/json, image/png
Connection: Keep-Alive
Content-Length: 65
Content-Type: application/json; charset=utf-8
Host: 127.0.0.1:4723
User-Agent: selenium/3.141.0 (.net windows)

{"using":"accessibility id","value":"<my_aid>"}
HTTP/1.1 200 OK
Content-Length: 101
Content-Type: application/json

{"sessionId":"<session_id>","status":0,"value":{"ELEMENT":"7.3440.63272130"}}


==========================================
```

The element is found and the test proceeds as normal.

However, the same step in my test on the Azure Pipeline test agent shows:

```
2021-02-05T17:10:36.4417628Z ==========================================
2021-02-05T17:10:36.4418839Z POST /session/<session_id>/element HTTP/1.1
2021-02-05T17:10:36.4420932Z 
2021-02-05T17:10:36.4422912Z Accept: application/json, image/png
2021-02-05T17:10:36.4424706Z 
2021-02-05T17:10:36.4427705Z Connection: Keep-Alive
2021-02-05T17:10:36.4428243Z 
2021-02-05T17:10:36.4431819Z Content-Length: 70
2021-02-05T17:10:36.4432107Z 
2021-02-05T17:10:36.4437295Z Content-Type: application/json; charset=utf-8
2021-02-05T17:10:36.4437649Z 
2021-02-05T17:10:36.4439233Z Host: 127.0.0.1:4723
2021-02-05T17:10:36.4440971Z 
2021-02-05T17:10:36.4442666Z User-Agent: selenium/3.141.0 (.net windows)
2021-02-05T17:10:36.4443085Z 
2021-02-05T17:10:36.4446953Z 
2021-02-05T17:10:36.4447307Z 
2021-02-05T17:10:36.4447650Z 
2021-02-05T17:10:36.4451068Z HTTP/1.1 404 Not Found
2021-02-05T17:10:36.4451348Z 
2021-02-05T17:10:36.4454257Z Content-Length: 139
2021-02-05T17:10:36.4454544Z 
2021-02-05T17:10:36.4457506Z Content-Type: application/json
2021-02-05T17:10:36.4459826Z 
2021-02-05T17:10:36.4460301Z 
2021-02-05T17:10:36.4462996Z 
2021-02-05T17:10:36.4465275Z {"status":7,"value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters."}}
2021-02-05T17:10:36.4498698Z 
2021-02-05T17:10:36.4501981Z 
2021-02-05T17:10:36.4505784Z ==========================================
```

The line that should be displaying the input JSON payload (the line above `HTTP/1.1 404 Not Found`) is blank. Is this test reporting that no element can be located because it's not receiving the input payload? If that's the case, why isn't it, when my local server receives it fine?

**Comment by mslukebo** [comment]
Update: it turns out, my application under test was crashing (for a completely unrelated reason) right before this `FindElement` invokation (but only on DevOps!)

It would be nice if WAD gave a different error besides "No element found" if the application is no longer running 😉
